toffy/watcher_callbacks.py

```diff
@@ -51,6 +54,25 @@ def plot_qc_metrics(self, qc_out_dir: str, **kwargs):
             qc_df = pd.read_csv(os.path.join(qc_out_dir, 'combined_%s.csv' % ms))
             visualize_qc_metrics(qc_df, metric_name, ax=axes[i], **viz_kwargs)
 
+    def plot_mph_metrics(self, mph_out_dir, img_out_dir, **kwargs):
```

Comment:
`plot_dir` would be more intuitive, img_out sounds like the actual images are being saved